src/builder.ts

```diff
@@ -275,6 +280,16 @@ export type DynamicOutputProperties = Record<
   DynamicOutputPropertyDef<string>
 >;
 
+export type TypeDef = AllNexusNamedTypeDefs | GraphQLNamedType;
+
+export type BlockDef =
```

Comment:
Seems correct to me. Nexus works with a superset concept of blocks of which type definitions are a subset.